hudi-common/src/main/java/org/apache/hudi/common/table/log/block/HoodieCommandBlock.java

```diff
@@ -36,7 +36,7 @@ public class HoodieCommandBlock extends HoodieLogBlock {
    * Hoodie command block type enum.
    */
   public enum HoodieCommandBlockTypeEnum {
-    ROLLBACK_PREVIOUS_BLOCK
+    ROLLBACK_BLOCK
```

Comment:
Is this backwards compatible? Making sure we dont persist this by name

Reply:
In order to preserve forward compatibility (say 0.8 generated log file, to be read by the latest version), we should keep ROLLBACK_PREVIOUS_BLOCK and introduce a new ROLLBACK_BLOCK.  Otherwise, we may run into upgrade issues (unable to read the older rollback block)